core/util/src/main/java/com/yapp/util/Extensions.kt

```
@@ -0,0 +1,4 @@
+package com.yapp.util
+
+class Extensions {
+}
```

detekt (1.23.8)

[warning] The class or object Extensions is empty.

(detekt.empty-blocks.EmptyClassBlock)